src/s3ql/backends/swiftks.py

```diff
@@ -76,18 +76,56 @@ def _get_conn(self):
             tenant = None
             user = self.login
 
-        auth_body = { 'auth':
+        if self.options.get('v3-auth', False):
+            domain = self.options.get('domain', 'Default')
```

Comment:
Can we get rid of "v3-auth" option and have any setting of "domain" imply that v3-auth should be used?